src/frontend/Komponenter/Behandling/VedtakOgBeregning/Skolepenger/Søknadsutgifter.tsx

```diff
@@ -0,0 +1,95 @@
+import { Søknadsgrunnlag } from '../../../../Felles/Ikoner/DataGrunnlagIkoner';
+import { Normaltekst } from 'nav-frontend-typografi';
+import React, { useMemo } from 'react';
+import DataViewer from '../../../../Felles/DataViewer/DataViewer';
+import { Ressurs } from '../../../../App/typer/ressurs';
+import { ISøknadsutgifterSkolepenger } from '../../../../App/typer/beregningssøknadsdata';
+import { useDataHenter } from '../../../../App/hooks/felles/useDataHenter';
+import { AxiosRequestConfig } from 'axios';
+import styled from 'styled-components';
+import { Heading, Label } from '@navikt/ds-react';
+import { formaterTallMedTusenSkilleEllerStrek } from '../../../../App/utils/formatter';
+
+const BoldTekst = styled(Label)`
+    margin-left: 0.25rem;
+`;
+
+const Container = styled.div`
+    width: 250px;
+`;
+
+const HøyrestiltTekst = styled(Normaltekst)`
+    display: flex;
+    justify-content: right;
+`;
+
+const IkonOgTekstWrapper = styled.div`
+    display: flex;
+    justify-content: flex-start;
+`;
+
+const Grid = styled.div`
+    display: grid;
+    grid-template-areas: 'utgiftstype utgiftsbeløp';
+    grid-template-columns: 8.5rem 5rem;
+    grid-gap: 0.5rem;
+    margin-bottom: 0.5rem;
+`;
+
+export const Søknadsutgifter: React.FC<{ behandlingId: string }> = ({ behandlingId }) => {
+    const søknadDataConfig: AxiosRequestConfig = useMemo(
+        () => ({
+            method: 'GET',
+            url: `/familie-ef-sak/api/soknad/${behandlingId}/utgifter-skolepenger`,
+        }),
+        [behandlingId]
+    );
+
+    const søknadDataResponse: Ressurs<ISøknadsutgifterSkolepenger> = useDataHenter<
+        ISøknadsutgifterSkolepenger,
+        null
+    >(søknadDataConfig);
+
+    const utledUtgiftsbeløp = (utgift?: number) => {
+        return utgift ? `${formaterTallMedTusenSkilleEllerStrek(utgift)} ,-` : '-';
+    };
+
+    return (
+        <DataViewer response={{ søknadDataResponse }}>
+            {({ søknadDataResponse }) => (
+                <Container>
+                    <Heading spacing size="small" level="5">
+                        Utgifter fylt inn i søknad
+                    </Heading>
+                    <Grid>
+                        <IkonOgTekstWrapper>
+                            <Søknadsgrunnlag />
+                            <BoldTekst size="small">Semesteravgift:</BoldTekst>
+                        </IkonOgTekstWrapper>
+                        <HøyrestiltTekst>
```

Comment:
Det er en del andre felter som man kunde vist her og, `hvorMyeSkalDuStudere`. En del som allerede finnes under vilkåret aktivitet. 
Er det noen grunn til at noe data vises på vilkår og noe her? (til funksjonelle)

Hadde man lagt allt på vilkåret/vilkårsgrunnlag (går fortsatt å vise her) så hadde vi ikke trengt en separat endepunkt i backend for å hente denne informasjonen

Reply:
Tipper at det kan være interessant, ja. Kan jo høre med Lars når han kommer